keyboards/crkbd/keymaps/devdev/config.h

```diff
@@ -0,0 +1,117 @@
+/*
+This is the c configuration file for the keymap
+
+Copyright 2012 Jun Wako <wakojun@gmail.com>
+Copyright 2015 Jack Humbert
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
+//#define SSD1306OLED
+#define CUSTOM_FONT
+
+#define CUSTOM_LAYER_READ //if you remove this it causes issues - needs better guarding 
+#define USE_SERIAL_PD2
+
+#define TAPPING_FORCE_HOLD
+#define TAPPING_TERM 200
+
+#define RGBLIGHT_SLEEP
+//
+#define RGBLIGHT_LAYERS
+
+#ifdef RGBLIGHT_ENABLE
+    #undef RGBLED_NUM
+	
+    //#define RGBLIGHT_ANIMATIONS
+	#define RGBLIGHT_EFFECT_BREATHING
+	#define RGBLIGHT_EFFECT_RAINBOW_MOOD
+	#define RGBLIGHT_EFFECT_RAINBOW_SWIRL
+	#define RGBLIGHT_EFFECT_SNAKE
+	#define RGBLIGHT_EFFECT_KNIGHT
+	//#define RGBLIGHT_EFFECT_CHRISTMAS
+	//#define RGBLIGHT_EFFECT_STATIC_GRADIENT
+	//#define RGBLIGHT_EFFECT_RGB_TEST
+	//#define RGBLIGHT_EFFECT_ALTERNATING
+	//#define RGBLIGHT_EFFECT_TWINKLE
+	
+    //#define RGBLED_NUM 54
+	//#define RGBLED_SPLIT 27
+	//#define RGBLED_SPLIT { 27, 27 } // haven't figured out how to use this yet 
```

Comment:
You need to use the "common" revision for this to work.

Additionally, hopefully the legacy version will be deprecated in the near future.

Reply:
Thanks, I just gave up and found alternative ways to go about having different displays on each side